mycompany-seeq-link-connector-myconnector/src/main/java/com/mycompany/seeq/link/connector/MyConnection.java

```diff
@@ -174,29 +186,49 @@ public void index(SyncMode syncMode) {
             // If you need the signals to be written to Seeq Server before any other work continues, you can
             // call flushSignals() on the connection service.
             this.connectionService.putSignal(signal);
+
+            ConditionUpdateInputV1 condition = new ConditionUpdateInputV1();
+
+            // The Data ID is a string that is unique within the data source, and is used by Seeq when referring
```

Comment:
We should probably explain that the DataID should be able to be consistent across connections. Meaning that transient values like Datasource name, or madeup GUIDs are bad choices for Data ID's.